google-services.json

```diff
@@ -0,0 +1,81 @@
+{
+  "project_info": {
+    "project_number": "754329551899",
+    "firebase_url": "https://dummy-showcase.firebaseio.com",
+    "project_id": "dummy-showcase",
+    "storage_bucket": "cordova-dummy.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:754329551899:android:621420eb6f5bc9df",
+        "android_client_info": {
+          "package_name": "com.aerogear.dummy"
```

Comment:
I believe this need to be changed to `org.aerogear.ionic.showcase` to match the app's package name, otherwise the android app won't build.